.github/workflows/devworkspace-generator-release.yml

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2020-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: Release Che Devworkspace Generator
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'release version in format 7.y.z'
+        required: true
+      forceRecreateTags:
+        description: If true, tags will be recreated. Use with caution
+        required: false
+        default: 'false'
+
+jobs:
+  build:
+    name: Create Che Devfile Registry Release
+    runs-on: ubuntu-22.04
+    steps:
+      -
+        name: "Checkout source code"
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - 
+        name: Check existing tags
+        if: github.event.inputs.performRelease == 'true'
+        run: |
+          set +e
+          RECREATE_TAGS=${{ github.event.inputs.forceRecreateTags }}
+          VERSION=${{ github.event.inputs.version }}
+          EXISTING_TAG=$(git ls-remote --exit-code origin refs/tags/${VERSION})
+          if [[ -n ${EXISTING_TAG} ]]; then
+            if [[ ${RECREATE_TAGS} == "true" ]]; then
+              echo "[INFO] Removing tag for ${VERSION} version. New tag will be recreated during release."
+              git push origin :$VERSION
+            else
+              echo "[ERROR] Cannot proceed with release - tag ${EXISTING_TAG} already exists."
+              exit 1
+            fi
+          else
+            echo "[INFO] No existing tags detected for $VERSION"
```

Comment:
if tag `$VERSION` doesn't exist, why don't we just create it here?

Reply:
it's a boilerplate code for "checking existence of tag, to avoid re-releasing existing tag, unless forced"

should we make attempt to tag the same repo in both workflows, or should we make one attempt in one of the workflows (right now it would be in latter 'che-devfile-registry' release)? To me, latter seems simpler